### PR TITLE
8 implement tmf f1 feature create team

### PR DESCRIPTION
**Pull request by asidun**
connect to  #8

### CODE REVIEW

File: src/test/resources/datasets/oneTeamInDBExists.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (petrokramar) [inline src/test/resources/datasets/oneTeamInDBExists.json:1]: oneTeamInDBExists.json equals to oneTeamInDB.json

File: src/test/java/juja/microservices/teams/service/TeamsServiceTest.java [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+        //Given
+        String expected = "SomeId";
+        when(teamRepository.add(any(Team.class))).thenReturn("SomeId");
+        //When
+        String result = teamService.addTeam(teamRequest);
+        //Then
+        assertEquals(expected, result);
+    }
+
+    @Test(expected = TeamUserExistsException.class)
```
Comment (petrokramar) [inline src/test/java/juja/microservices/teams/service/TeamsServiceTest.java:54]: test doesn't throw exception

**Comment by petrokramar** [comment]
Please, replace in .travis.yml 
- mongo teams --eval 'db.createUser({user:"root", pwd:"root", roles:[{role:"dbAdmin", db:"users"}]});'
with
- mongo teams --eval 'db.createUser({user:"root", pwd:"root", roles:[{role:"dbAdmin", db:"teams"}]});'

**Comment by petrokramar** [comment]
Add empty test into TeamsAcceptanceTest

File: src/test/resources/datasets/oneTeamInDBExists.json (hunk 1 above)
Comment (asidun) [reply src/test/resources/datasets/oneTeamInDBExists.json:1]: dismissDate is different ;)

File: src/test/java/juja/microservices/teams/service/TeamsServiceTest.java (hunk 2 above)
Comment (asidun) [reply src/test/java/juja/microservices/teams/service/TeamsServiceTest.java:54]: Should throw after last commit

**Review by petrokramar — APPROVED** [review]

File: src/test/java/juja/microservices/teams/service/TeamsServiceTest.java [hunk 3]
```diff
@@ -1,7 +1,61 @@
...
+/**
+ * @author Andrii.Sidun
+ */
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(TeamService.class)
 public class TeamsServiceTest {
 
+    @Inject
+    @InjectMocks
```
Comment (prfct) [inline src/test/java/juja/microservices/teams/service/TeamsServiceTest.java:28]: useless annotation in this case. should be @Inject + @MockBean or @Mock + @InjectMocks

**Pull request merged**